src/ConnectedMode/UI/ServerSelection/ServerSelectionWnd.xaml

```diff
@@ -0,0 +1,126 @@
+                         FontFamily="{DynamicResource {x:Static vsShell:VsFonts.EnvironmentFontFamilyKey}}"
+                         x:Name="This">
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="pack://application:,,,/SonarLint.VisualStudio.ConnectedMode;component/UI/Resources/Styles.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid DataContext="{Binding ElementName=This, Path=ViewModel}" Margin="10">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
+        </Grid.RowDefinitions>
+
+        <GroupBox Grid.Row="0" Header="{x:Static ui:UiResources.ChooseConnectionTypeLbl}">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+
+                <Grid Grid.Row="0" Margin="10">
+                    <Grid.RowDefinitions>
+                        <RowDefinition />
+                        <RowDefinition />
+                        <RowDefinition />
+                    </Grid.RowDefinitions>
+
+                    <StackPanel Grid.Row="0" Orientation="Horizontal" VerticalAlignment="Center">
+                        <RadioButton GroupName="ServerSelection" VerticalContentAlignment="Center" Margin="0,0,10,0" IsChecked="{Binding Path=IsSonarCloudSelected}"/>
+                        <Image Source="../Resources/SonarCloud.png" Width="250"/>
+                    </StackPanel>
+                    <TextBlock Grid.Row="1" Text="{x:Static ui:UiResources.SonarCloudDescriptionLbl}" Margin="10"/>
+                    <TextBlock Grid.Row="2" Margin="10,0">
+                        <Hyperlink NavigateUri="https://www.sonarsource.com/products/sonarcloud/" RequestNavigate="ViewWebsite">SonarCloud</Hyperlink>
+                        <Run Text="{x:Static ui:UiResources.SonarCloudFree}"></Run>
+                    </TextBlock>
+                </Grid>
+
+                <Grid Grid.Row="1" Margin="10">
+                    <Grid.RowDefinitions>
+                        <RowDefinition />
+                        <RowDefinition />
+                        <RowDefinition />
+                        <RowDefinition />
+                    </Grid.RowDefinitions>
+
+                    <StackPanel Grid.Row="0" Orientation="Horizontal" VerticalAlignment="Center">
+                        <RadioButton GroupName="ServerSelection" VerticalContentAlignment="Center" Margin="0,10,10,0" IsChecked="{Binding Path=IsSonarQubeSelected}"/>
+                        <Image Source="../Resources/SonarQube.png" Width="250"/>
```

Comment:
In dark themes, the image is getting lost in the background.

Is it possible to use the light version of the images when a dark theme is used? (Same as IntelliJ New connection)

Reply:
I moved the comment as part of this [UI task](https://sonarsource.atlassian.net/browse/SLVS-1340)